PULL_REQUEST_MANAGEMENT.md

```diff
@@ -0,0 +1,14 @@
+# Rules for managing Pull Requests
+
+For the sake of trying to maintain an acceptable number of open but idle PRs, the following rules should be considered:
+- If a PR has at least one approval, it can be merged after 1 week of waiting for additional comments.
+  - If you consider the change trivial enough or it has at least 3 approvals, it may also be merged right away.
+- If a PR stays in "changes requested" for 2 weeks, and that change has not been done in the meantime, it shall be closed.
+- If you require a review from a particular person, assign the PR to that person. Don't just rely on the "review requested" feature of GitHub.
+- Don't feel obliged to comment everything you see, just for the sake of commenting. Be it on JIRA, GitHub, or even on IRC.
```

Comment:
That doesn't sound like a Pull Request management thing. Maybe the title needs to be adjusted?